### ISSUE TITLE
No color visible in Cmder (ConEmu)

### ISSUE THREAD

**Issue by Murazaki** [romkatv/powerlevel10k#152]
Can't display colors on ConEmu, Cmder, Mintty, even via SSH on Windows.

I tried a similar configuration on iterm2 on macOS and it works well there.

ssh gives a specific error :
```
_p9k_get_icon:13: character not in range
```
![Screenshot](https://i.imgur.com/ms03LNG.png)

**Comment by Murazaki** [comment]
Ok sorry, fixed the Cmder part with this task :

```
set "PATH=C:\tools\cygwin\bin;%PATH%"

*%ConEmuBaseDirShort%\conemu-cyg-64.exe /usr/bin/zsh --login -i -new_console:p:C:"C:\tools\cygwin\Cygwin.ico" -cur_console:h0
```

Still not working with SSH

**Comment by romkatv** [comment]
The problem is that the locale in your SSH session isn't using utf-8 encoding. If you try to print a non-ascii character with `print '\u2B50'` or `echo $'\u2B50'`, you'll get "character not in range" error. It's not affecting just p10k but everything you run in the SSH session. To fix this problem you need to run `locale -a`, pick a locale that has `utf8`, `utf-8` or `UTF8` or `UTF-8` in its name, and export `LANG` from your `~/.zshrc` or `~/.zshenv` with the value set to the chosen locale.

For example:

```zsh
echo 'export LANG=C.UTF-8' >>! ~/.zshenv
```

Or:

```zsh
echo 'export LANG=en_US.UTF-8' >>! ~/.zshenv
```

I've update powerlevel10k so that it works when your terminal actually supports utf-8 but your locale says that it doesn't. If you update powerlevel10k and generate a new config with `p10k configure`, it'll work even with a misconfigured locale. However, you really need to fix your locale to avoid issues with all programs you'll run.

**Comment by Murazaki** [comment]
Thanks for the info.

Maybe we should add those details to the troubleshooting section ?

**Comment by romkatv** [comment]
Before I committed https://github.com/romkatv/powerlevel10k/commit/59e96984a1cde0c50a1c37e4b8ed9fe7b028b3c4 I had another commit that would print a warning if the current locale uses single-byte encoding. However, after I made p10k work in this environment, it's difficult to justify printing this warning. Yes, the SSH environment is broken, but it's not affecting p10k. If p10k is justified in printing a warning about some issue that doesn't affect it, does it mean that every program you run (zsh, ls, grep, etc.) should print such warning? Should p10k also print a warning if it sees that your `~/.ssh` is world-readable? Certainly a common problem and the consequences are bad.

In short, yes, I know that lots of users have misconfigured SSH sessions (this is because the default machine locale is usually not set for SSH connections), but it doesn't seems like it's the job of p10k to detect this issue and to nag people about it.

**Comment by Murazaki** [comment]
I was just talking about documentation on the Readme.md.

**Comment by romkatv** [comment]
> I was just talking about documentation on the Readme.md.

Same reasons apply to `README.md`.

**Comment by romkatv** [comment]
Could you confirm that your problem is now resolved?

**Comment by Murazaki** [comment]
Resolved
